Exercises-4-OOP/src/inheritanceanimal/Cat.java

```diff
@@ -0,0 +1,29 @@
+package inheritanceanimal;
+
+public class Cat extends Mammal {
+
+	 String greets = "meow";
+
+	public Cat() {
+
+	}
+
+	public Cat(Mammal mammal1, String greets) {
```

Comment:
This constructoris not part of the requirements, should be removed.

Reply:
if I remove it, the application won't run :(